Core/include/Acts/Propagator/detail/JacobianEngine.hpp

```diff
@@ -132,23 +114,18 @@ BoundToFreeMatrix boundToFreeTransportJacobian(
 /// This is an approximated approach to treat the (assumed) small change.
 ///
 /// @param [in] geoContext The geometry Context
+/// @param [in] surface The target surface
 /// @param [in] freeParameters Free, nominal parametrisation
-/// @param [in] directionToAnglesJacobian The relation jacobian from dir to
-/// angle
-/// @param [in] anglesToDirectionJacobian The relation jacobian from angle to
-/// dir
 /// @param [in] freeTransportJacobian Transport jacobian free to free
-/// @param [in] freeToPathDerivatives Path length derivatives for free
-/// parameters
-/// @param [in] surface The target surface
+/// @param [in] freeToPathDerivatives Path length derivatives for free parameters
+/// @param [out] fullTransportJacobian The 6x8 transport jacobian from bound to free
 ///
-/// @return the 6x8 transport jacobian from bound to free
-FreeToBoundMatrix freeToBoundTransportJacobian(
-    const GeometryContext& geoContext, const FreeVector& freeParameters,
-    const ActsMatrix<7, 8>& directionToAnglesJacobian,
-    const ActsMatrix<8, 7>& anglesToDirectionJacobian,
-    const FreeMatrix& freeTransportJacobian,
-    const FreeVector& freeToPathDerivatives, const Surface& surface);
+void freeToBoundTransportJacobian(const GeometryContext& geoContext,
+                                  const Surface& surface,
+                                  const FreeVector& freeParameters,
+                                  const FreeMatrix& freeTransportJacobian,
```

Comment:
`freeTransportJacobian` is the contraction of the direction to angles and angles to direction jacobians?

Reply:
I think this is basically what we get from the stepper. the full jacobian from point A to point B in free coordinates

Reply:
oh I see. I think these direction to angles and vice versa are done only by the global to local now. looks like there was a different free representation once with angles instead of directions